app/views/public_activity/raw_pending_stripe_transaction/_create.html.erb

```diff
@@ -4,19 +4,19 @@
 <% if hcb_code %>
   <% if hcb_code.stripe_refund? %>
     <%= render layout: "/public_activity/common", locals: { activity:, current_user:, url: hcb_code_path(hcb_code) } do %>
-      was refunded <%= render_money hcb_code.amount_cents.abs %> from <i><%= hcb_code.memo %></i>
+      was refunded <%= render_money hcb_code.amount_cents.abs %> from <i><%= hcb_code.memo %></i> for <%= link_to hcb_code.event&.name, hcb_code.event %>
     <% end %>
   <% elsif hcb_code.pt&.declined? %>
     <%= render layout: "/public_activity/common", locals: { possessive: true, current_user:, activity:, url: hcb_code_path(hcb_code) } do %>
-      card was declined for <%= render_money activity.trackable.amount_cents.abs %> at <i><%= hcb_code.memo %></i>
+      card was declined for <%= render_money activity.trackable.amount_cents.abs %> at <i><%= hcb_code.memo %></i> for <%= link_to hcb_code.event&.name, hcb_code.event %>
```

Comment:
I think these are becoming run on sentences. Maybe we make it Sam's Hack Club HQ card was declined X for X

Reply:
Open to other ideas

Reply:
I like that!